e2e/tests/plugins/condition/condition.e2e.spec.js

```diff
@@ -32,39 +32,40 @@ const { expect } = require('@playwright/test');
 let conditionSetUrl;
 let getConditionSetIdentifierFromUrl;
 
-test('Create new Condition Set object and store @localStorage', async ({ page, context }) => {
-    //Go to baseURL
-    await page.goto('/', { waitUntil: 'networkidle' });
-
-    //Click the Create button
-    await page.click('button:has-text("Create")');
-
-    // Click text=Condition Set
-    await page.click('text=Condition Set');
+test.describe.serial('Condition Set CRUD Operations on @localStorage', () => {
```

Comment:
had to refactor this test to use the new test. also seemed like it was occasionally failing as the sibling test appeared to be running async of the serial tests below (which depended on the sibling test finishing)